pkg/generated/openapi/cmd/models-schema/main.go

```diff
@@ -19,12 +19,11 @@ package main
 import (
 	"encoding/json"
 	"fmt"
-	"os"
-	"strings"
-
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
was this on purpose before to avoid this dependency?

cc: @liggitt

Reply:
not sure what the original intent was, but we do already have this indirectly via k8s.io/kubernetes/pkg/generated/openapi → k8s.io/apimachinery/pkg/apis/meta/v1 → k8s.io/apimachinery/pkg/runtime

Reply:
Previously there were 4 private copies of the [ToRESTFriendlyName](https://github.com/kubernetes/kube-openapi/blob/c8a335a9a2ffc5aff16dfef74896a1ee34eb235d/pkg/util/util.go#L58) function, all named `friendyName`.  This reduces that down to one in k8s.io/apimachinery/pkg/runtime.

I kept the last one to avoid adding a dependency from k8s.io/apimachinery → k8s.io/kube-openapi which results dependencies like k8s.io/api → k8s.io/apimachinery → k8s.io/kube-openapi. I didn't want to add that.

Here's the last remaining copy:

https://github.com/kubernetes/kubernetes/pull/131616/files#diff-7ac25bcc6f0498d4ade98f19cfd251d2cb4cdfbe3c21a72eb7ba8eb0b0f4fdd0R771-R773